packages/ppf-contracts/contracts/PPF.sol

```diff
@@ -47,27 +47,67 @@ contract PPF is IFeed, TimeHelpers {
         _setOperatorOwner(_operatorOwner);
     }
 
+    /**
+    * @notice Set operator public key to `_operator`
+    * @param _operator Public key allowed to sign messages to update the pricefeed
+    */
+    function setOperator(address _operator) external {
+        // Allow the current operator to change the operator to avoid having to hassle the
+        // operatorOwner in cases where a node just wants to rotate its public key
+        require(msg.sender == operator || msg.sender == operatorOwner, ERROR_CAN_NOT_SET_OPERATOR);
+        _setOperator(_operator);
+    }
+
+    /**
+    * @notice Set operator owner to `_operatorOwner`
+    * @param _operatorOwner Address of an account that can change the operator
+    */
+    function setOperatorOwner(address _operatorOwner) external {
+        require(msg.sender == operatorOwner, ERROR_CAN_NOT_SET_OPERATOR_OWNER);
+        _setOperatorOwner(_operatorOwner);
+    }
+
+    /**
+    * @param base Address for the base token in the feed
+    * @param quote Address for the quote token the base is denominated in
+    * @return XRT for base:quote and the timestamp when it was updated
+    */
+    function get(address base, address quote) public view returns (uint128, uint64) {
```

Comment:
Generally we put `view` functions after non-view functions.

We normally have a linting rule set up for this; I'm surprised we haven't set up linting for ppf at all 😅

Reply:
sure, let's do that :)

Reply:
actually this one is an external function, we are overriding its visibility :/